R/PackageFunctionReporter.R

```diff
@@ -209,7 +210,12 @@ FunctionReporter <- R6::R6Class(
             # Filter objects in package environment to just functions
             # This will now be a character vector full of function names
             funs <- Filter(
-                f = function(x, p = pkg_env){is.function(get(x, p))}
+                f = function(x, p = pkg_env){
+                    (is.function(get(x, p))
+                        # Exclude Reference Class object generators for now
+                        & !methods::is(get(x, p), "refObjectGenerator")
```

Comment:
WOW good catch, nice nice.

Out of curiosity, why was it necessary to add these wrapping `()`?

Reply:
``` r
TRUE &
    FALSE
#> [1] FALSE
```

``` r
TRUE
    & FALSE
#> Error: <text>:2:5: unexpected '&'
#> 1: TRUE
#> 2:     &
#>        ^
```

``` r
(TRUE
    & FALSE    
)
#> [1] FALSE
```

<sup>Created on 2018-11-30 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>